deployments/warp_routes/USDT/arbitrum-ethereum-mantle-mode-polygon-scroll-zeronetwork-config.yaml

```diff
@@ -21,12 +16,7 @@ tokens:
     coinGeckoId: tether
     collateralAddressOrDenom: "0x201EBa5CC46D216Ce6DC03F6a759e8E766e956aE"
     connections:
-      - token: ethereum|arbitrum|0x83AF845F002c2dA8748eaB8203D2704BdAE6fB8e
-      - token: ethereum|mode|0x803e7524526c579cCF6Ef0474d03012EdFD0d3Ec
-      - token: ethereum|polygon|0x6701d503369cf6aA9e5EdFfEBFA40A2ffdf3dB21
-      - token: ethereum|scroll|0x0FC41a92F526A8CD22060A4052e156502D6B9db0
       - token: ethereum|zeronetwork|0x36dcfe3A0C6e0b5425F298587159249d780AAfab
-      - token: ethereum|ethereum|0x95fBD1037a307A49587174E06a6600fD05c5f58e
```

Comment:
iiuc, they're there atm but won't be after the multisig tx lands

Reply:
Nearly all of them are not connected anymore after the tx from the 10th of December the only one that still is, is mantle

Reply:
can confirm, did `warp read` on all these routes and compared enrolled routers for each chain 😅